affiliatewp-order-details-for-affiliates.php

```diff
@@ -349,24 +385,25 @@ public function global_order_details_access() {
 	}
 
 	/**
-	 * Modify plugin metalinks
+	 * Modify plugin metalinks.
```

Comment:
```suggestion
	 * Modifies the plugin metalinks.
```
Third-person singular verb in a DocBlock summary.